wgpu-core/src/device/global.rs

```diff
@@ -2224,6 +2224,15 @@ impl Global {
         }
     }
 
+    // This is a test-only function to force the device into an
+    // invalid state by inserting an error value in its place in
+    // the registry.
+    pub fn device_make_invalid<A: HalApi>(&self, device_id: DeviceId) {
```

Comment:
```suggestion
    /// This is a test-only function to force the device into an
    /// invalid state by inserting an error value in its place in
    /// the registry.
    #[cfg(test)]
    pub fn device_make_invalid<A: HalApi>(&self, device_id: DeviceId) {
```

Reply:
@teoxoy: This feedback (and others like it) couldn't ever work with an external test crate like `wgpu-test`. `wgpu-test` uses the public API of `wgpu`…so we need a different option.

In the most recent `wgpu maintainers meeting`, we [resolved](https://docs.google.com/document/d/1I3qh1kShDu48VehgFarftEpbEqZK8e8kWqaDo05e1jM/edit#heading=h.4boukitzny7l) to try just using `#[doc(hidden)]`. I've added this attribute to the suggested functions instead of `cfg(test)` with 76eec146f, which I think should resolve this and other feedback.